.github/workflows/ios-ci.yml

```diff
@@ -29,6 +29,7 @@ on:
 permissions:
   id-token: write         # needed for AWS
   contents: write         # allow making a release
+  pages: write
```

Comment:
Nit: Could add a comment like the other permissions so we know what this permission is needed for

Reply:
Will add it in a follow-up.